txscript/standard.go

```diff
@@ -720,26 +718,6 @@ func MultisigRedeemScriptFromScriptSig(script []byte) []byte {
 	return finalOpcodeData(scriptVersion, script)
 }
 
-// GenerateSSGenBlockRef generates an OP_RETURN push for the block header hash and
-// height which the block votes on.
-func GenerateSSGenBlockRef(blockHash chainhash.Hash, height uint32) ([]byte, error) {
-	// Serialize the block hash and height
-	brBytes := make([]byte, 32+4)
-	copy(brBytes[0:32], blockHash[:])
-	binary.LittleEndian.PutUint32(brBytes[32:36], height)
-
-	return NewScriptBuilder().AddOp(OP_RETURN).AddData(brBytes).Script()
-}
-
-// GenerateSSGenVotes generates an OP_RETURN push for the vote bits in an SSGen tx.
-func GenerateSSGenVotes(votebits uint16) ([]byte, error) {
-	// Serialize the votebits
-	vbBytes := make([]byte, 2)
-	binary.LittleEndian.PutUint16(vbBytes, votebits)
-
-	return NewScriptBuilder().AddOp(OP_RETURN).AddData(vbBytes).Script()
-}
-
```

Comment:
Not important for the end product, but this is removed in the wrong commit I believe. It should be in the one after this.

Reply:
Yep, looks like a rebase snafu.  Thanks!  Fixed.